cluster-test/src/main/java/com/xjeffrose/chicago/ChicagoClientTest.java

```diff
@@ -0,0 +1,270 @@
+package com.xjeffrose.chicago;
+
+import com.google.common.primitives.Longs;
+import com.xjeffrose.chicago.ChicagoMessage;
+import com.xjeffrose.chicago.ChicagoObjectDecoder;
+import com.xjeffrose.chicago.client.BaseChicagoClient;
+import com.xjeffrose.chicago.client.ChicagoClient;
+import com.xjeffrose.chicago.client.ChicagoClientException;
+import com.xjeffrose.chicago.client.ChicagoClientTimeoutException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertNotNull;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by root on 8/8/16.
+ */
+public class ChicagoClientTest {
+  TestChicagoCluster testChicagoCluster;
+  HashMap<String, String> servers = new HashMap<>();
+
+  @Before
+  public void setup() throws Exception {
+    servers.put("10.24.25.188:12000", "10.24.25.188:12000");
```

Comment:
This is the case where you would put an application.conf into src/test/resources